api/acl.go

```diff
@@ -45,6 +45,10 @@ type ACLToken struct {
 	// DEPRECATED (ACL-Legacy-Compat)
 	// Rules will only be present for legacy tokens returned via the new APIs
 	Rules string `json:",omitempty"`
+
+	// Namespace is the namespace the ACLToken is associated with.
+	// Namespacing is a Consul Enterprise feature.
```

Comment:
```suggestion
	// Namespaces is a Consul Enterprise feature.
```

nit: this is what the error message says.